srcpkgs/gajim-plugin_installer/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'gajim-plugin_installer'
+pkgname=gajim-plugin_installer
+_gajim_version=1.3
+version=1.3.8
```

Comment:
```suggestion
version=1.3.8
_gajim_version=${version%.*}
```

That said, where does `version` come from, given that we are using only `_gajim_version` to download the distfile?

Reply:
It looks like, upstream uses this naming scheme. So I see no reason to not implement `${version%.*}` in the url. Imho everything is clear with this.

Reply:
https://github.com/void-linux/void-packages/pull/28982#issuecomment-794646320